commons/zenoh-keyexpr/src/key_expr/tests.rs

```diff
@@ -151,7 +151,7 @@ fn inclusions() {
 #[test]
 fn fuzz() {
     const FUZZ_ROUNDS: usize = 100_000;
-    let rng = rand::thread_rng();
+    let rng = rand::rngs::OsRng::default();
```

Comment:
I'm curious about this change. Why replacing `thread_rng()` with `OsRng::default()`?

Reply:
`thread_rng()` is available only in `std`. I think what we preciously did in similar situations is add a `test` feature that enables `std` in cascade, and have `rand` methods be enabled only when `test` is enabled.

Reply:
like davide explained, except I rolled it back and missed this instance because it got lost in all the file moves ^^"

Reply:
Not a big deal for me. Up to you wether to roll it back to `thread_rng`.